app/src/androidTestWebview/java/org/mozilla/focus/activity/SearchEngineSelectionTest.java

```diff
@@ -60,85 +74,140 @@ public void tearDown() throws Exception {
-
-        GoogleSelection.waitForExists(waitingTime);
-        GoogleSelection.click();
-        TestHelper.pressBackKey();
-        // Now it's changed to Google
-        assertTrue(searchName.getText().equals("Google"));
-        TestHelper.settingsHeading.waitForExists(waitingTime);
-        TestHelper.pressBackKey();
-
-        /* load blank spaces and press enter key for search, it should not do anything */
-        TestHelper.inlineAutocompleteEditText.waitForExists(waitingTime);
-        TestHelper.inlineAutocompleteEditText.clearTextField();
-        TestHelper.inlineAutocompleteEditText.setText("   ");
-        TestHelper.pressEnterKey();
-        assertTrue(TestHelper.inlineAutocompleteEditText.exists());
-
-        /* Now do some search */
-        TestHelper.inlineAutocompleteEditText.clearTextField();
-        TestHelper.inlineAutocompleteEditText.setText("mozilla focus");
-        TestHelper.hint.waitForExists(waitingTime);
-
-        // Check the search hint bar is correctly displayed
-        assertTrue(TestHelper.hint.getText().equals("Search for mozilla focus"));
-        TestHelper.hint.click();
-
-        /* Browser shows google search webview*/
-        googleWebView.waitForExists(waitingTime);
-        TestHelper.progressBar.waitForExists(webPageLoadwaitingTime);
-        Assert.assertTrue(TestHelper.progressBar.waitUntilGone(webPageLoadwaitingTime));
-        assertTrue (TestHelper.browserURLbar.getText().contains("google"));
-        assertTrue (TestHelper.browserURLbar.getText().contains("mozilla"));
-        assertTrue (TestHelper.browserURLbar.getText().contains("focus"));
-
-        /* tap url bar, and check it displays search term instead of URL */
-        TestHelper.browserURLbar.click();
-        TestHelper.inlineAutocompleteEditText.waitForExists(waitingTime);
-        assertEquals(TestHelper.inlineAutocompleteEditText.getText(), "mozilla focus");
-        TestHelper.pressEnterKey();
-        googleWebView.waitForExists(waitingTime);
-        TestHelper.progressBar.waitForExists(webPageLoadwaitingTime);
-        Assert.assertTrue(TestHelper.progressBar.waitUntilGone(webPageLoadwaitingTime));
-        assertTrue (TestHelper.browserURLbar.getText().contains("google"));
-        assertTrue (TestHelper.browserURLbar.getText().contains("mozilla"));
-        assertTrue (TestHelper.browserURLbar.getText().contains("focus"));
-
-         /* Now do another search */
-        TestHelper.browserURLbar.click();
-        TestHelper.inlineAutocompleteEditText.waitForExists(waitingTime);
-        TestHelper.inlineAutocompleteEditText.clearTextField();
-        TestHelper.inlineAutocompleteEditText.setText("mozilla focus");
-        TestHelper.hint.waitForExists(waitingTime);
-
-        // Check the search hint bar is correctly displayed
-        assertTrue(TestHelper.hint.getText().equals("Search for mozilla focus"));
-        TestHelper.hint.click();
+
+        // Open [settings menu] and select Search
+        {
```

Comment:
we probably won't need open/close braces for each section, have them separated by an empty line should be enough, with the comment on top

Reply:
thanks @npark-mozilla, i saw several test cases with this formatting.  wasn't sure if we were adhereing to a given standard